### DISCUSSION TITLE
How to disable jumping?

### DISCUSSION THREAD

**Discussion by gaoqiangks** [kylechui/nvim-surround#250] [Q&A] [answered]
If the cursor is not located inside a surrounding pair, it can jump to the "nearest" pair.  This is an excellent feature.  However, if I mistyped  something accedently,  the cursor may jump to somewhere I don't mean to be, and it would change my code. In this case, I have carefully check what has been changed.   So, is there any way to disable this feature?

**Comment by kylechui** [comment, marked as answer]
See `:h nvim-surround.config.move_cursor`. Note that #112 is still an active issue.

**Reply by gaoqiangks** [reply]
I found that this is a vim issue and it's the default behavior of vim. Thank you.
